src/v2/view-builder/views/webauthn/ChallengeWebauthnView.js

```diff
@@ -20,9 +20,10 @@ const Body = BaseForm.extend({
     if (webauthn.isNewApiAvailable()) {
       const retryButton = createButton({
         className: 'retry-webauthn button-primary default-custom-button',
-        title: loc('retry', 'login'),
+        title: loc('mfa.challenge.verify', 'login'),
         click: () => {
           this.getCredentialsAndSave();
```

Comment:
should we add the webauthn api check here? `(webauthn.isNewApiAvailable())`

Reply:
we have it here https://github.com/okta/okta-signin-widget/pull/1580/files/441c9e9151e56eb3194677ec947bfb6b781fdcaa#diff-3b37256e539729ec15206125fe750fa4e6bf15ebdcd7854ee9a5737123d827a7L20

Reply:
ahh yes! 👍